packages/components/vitest.config.ts

```diff
@@ -54,6 +52,15 @@ export default defineConfig(({ mode }) => ({
         'src/assets/**',
         'src/styles/**',
         'src/types/**',
+        '**/*.query.ts',
+        'src/edge-runtime-compatible/**',
```

Comment:
Same as in https://github.com/City-of-Helsinki/events-helsinki-monorepo/pull/823/files#r2336917246 i.e. why should this folder be excluded?